Plugins/MatrixPlugin/MatrixForm.cs

```diff
@@ -0,0 +1,257 @@
+                rbOverSleaze.DoDataBinding("Enabled", logic, "OverClocker");
+                rbOverDataProc.DoDataBinding("Enabled", logic, "OverClocker");
+                rbOverFirewall.DoDataBinding("Enabled", logic, "OverClocker");
+                AddRadioCheckedBinding(rbOverAttack, logic, "Overclocked", "Attack");
+                AddRadioCheckedBinding(rbOverSleaze, logic, "Overclocked", "Sleaze");
+                AddRadioCheckedBinding(rbOverDataProc, logic, "Overclocked", "DataProcessing");
+                AddRadioCheckedBinding(rbOverFirewall, logic, "Overclocked", "Firewall");
+
+                lAttackMod.DoDataBinding("Text", logic, "AttackMod");
+                lSleazeMod.DoDataBinding("Text", logic, "SleazeMod");
+                lDataProcMod.DoDataBinding("Text", logic, "DataProcessingMod");
+                lFirewallMod.DoDataBinding("Text", logic, "FirewallMod");
+
+                lAttackRes.DoDataBinding("Text", logic, "TotalAttack");
+                lSleazeRes.DoDataBinding("Text", logic, "TotalSleaze");
+                lDataProcRes.DoDataBinding("Text", logic, "TotalDataProcessing");
+                lFirewallRes.DoDataBinding("Text", logic, "TotalFirewall");
+                //Action section
+                lSkillDescription.DoDataBinding("Text", logic, "currentAction.Description");
+                lActionType.DoDataBinding("Text", logic, "currentAction.Type");
+                lActionAttributeName.DoDataBinding("Text", logic, "currentAction.ActionAttribute");
+                lActionSkillName.DoDataBinding("Text", logic, "currentAction.ActionSkill");
+                lSkillLimitName.DoDataBinding("Text", logic, "currentAction.Limit");
+                lDefendAttributeName.DoDataBinding("Text", logic, "currentAction.DefenceAttribute");
+                lDefendSkillName.DoDataBinding("Text", logic, "currentAction.DefenceSkill");
+
+                DoDataBindingWithFormatter(lActionAttributeValue, "currentAction.ActionAttribute", new ConvertEventHandler(AttributeToValue));
+                DoDataBindingWithFormatter(lActionSkillValue, "currentAction.ActionSkill", new ConvertEventHandler(SkillToValue));
+                DoDataBindingWithFormatter(lSkillLimitValue, "currentAction.Limit", new ConvertEventHandler(MatrixAttributeToValue));
+                DoDataBindingWithFormatter(lDefendAttributeValue, "currentAction.DefenceAttribute", new ConvertEventHandler(AttributeToValue));
+                DoDataBindingWithFormatter(lDefendSkillValue, "currentAction.DefenceSkill", new ConvertEventHandler(MatrixAttributeToValue));
+                lActionModifier.DoDataBinding("Text", logic, "currentAction.ActionModifier");
+                lDefendModifier.DoDataBinding("Text", logic, "currentAction.DefenceModifier");
+                dpcActionDicePool.DoDataBinding("DicePool", logic, "ActionDicePool");
+                dpcDefendDicePool.DoDataBinding("DicePool", logic, "DefenceDicePool");
```

Comment:
One, try to use `nameof()` to make future refactoring easier. Two, I don't think you added `OnPropertyChanged()` hooks to `MatrixLogic::Overclocked`, so a lot of these bindings are going to misbehave.

Reply:
Edit it in https://github.com/chummer5a/chummer5a/pull/4563/commits/626b1cbbfe4b1451202ee7d326677b14f7bd6cc9